roles/trento_rpms/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+# code: language=ansible
+---
+- name: Create secrets
+  no_log: true
+  ansible.builtin.set_fact: # noqa: var-naming[no-jinja]
```

Comment:
Not you fault, or nothing related to this particular PR, but at some point I and @CDimonaco discussed that we shouldn't create the secrets randomly, and that we should always ask them to the user.
I don't know where that conversion happened...
Maybe we should open a tech-debt ticket for that